lib/kses.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * KSES related functions for the Gutenberg plugin.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds attributes to kses allowed tags that aren't in the default list
+ * and that Gutenberg needs to save blocks such as the Gallery block.
+ *
+ * @param array $tags Allowed HTML.
+ * @return array (Maybe) modified allowed HTML.
+ */
+function gutenberg_kses_allowedtags( $tags ) {
+	$tags['a']['download']      = true;
```

Comment:
Something the previous implementation had accounted for which this does not is if a developer calls to `unset( $tags['a'] );` in their own filter handler before this one is called. Here, I think an error may occur.